README.md

```diff
@@ -1,23 +1,35 @@
 # DBGenerator
 
-Cet outil permet la génération de modèle de base de données pour les plateformes Android et iOS à partir d'un fichier xcdatamodel. Les SGBD supportés pour le moment sont les suivants :
+DBGenerator is a tool that generates Realm.io classes, for both Android and iOS, from a `.xcdatamodel` file.
+<!--Cet outil permet la génération de modèle de base de données pour les plateformes Android et iOS à partir d'un fichier xcdatamodel. Les SGBD supportés pour le moment sont les suivants :-->
 
 * Realm
 
 ---
 
 
+## What is xcdatamodel
+The `.xcdatamodel` is an XML file, generated by [Xcode](https://en.wikipedia.org/wiki/Xcode). It encodes the CoreData entities and can be created or edited in a graphical user interface.

```

Comment:
Keep an empty line between section titles and content (as some Markdown editors are not as lenient as GitHub and require a blank line to parse it properly)